docs/getting_started/intellij_plugin.md

```diff
@@ -0,0 +1,96 @@
+This plugin provides support for working with Chutney scenarios files in IntelliJ IDEA based IDEs.
+# Installation
+
+* Download chutney-idea-plugin-X.Y.zip from [available releases](https://github.com/chutney-testing/chutney-idea-plugin/releases){:target=_blank}.
+* In your Intellij Ide, go to File -> Settings -> Plugins.
+* Click install plugin from disc and select the downloaded zip.
+  ![plugin installation](../img/idea_plugin/install-plugin.gif)
+* Restart your IDE.
+
+
+
+# Configuration
+Configuration files can be found in the [intellij_config_folder](https://intellij-support.jetbrains.com/hc/en-us/articles/206544519-Directories-used-by-the-IDE-to-store-settings-caches-plugins-and-logs#:~:text=Configuration%20(idea.config.path)%3A%20%25APPDATA%25%5CJetBrains%5CIntelliJIdea2022.2){:target=_blank}`/chutney-idea-plugin/conf` folder.
+### Define an environment
+An environment named `GLOBAL` must be created in `intellij_config_folder/conf/GLOBAL.json`. It contains all targets used in scenarios. </br>
+You can get access to this file directly from your IDE
+![config file](../img/idea_plugin/config_file.png)
+
+You can find [here](/getting_started/write/#declare-an-environment) an example of how to declare an environment using JSON.
+
+# Features
+!!! important
+    To be recognized by the plugin, a scenario's json file must be suffixed with `.chutney.json`.
+    This file can be generated automatically by the [synchronize](/installation/ci_cd/scenario_sync/) feature.
+
+### Completion
```

Comment:
I prefer to have sub-menu as `##` because each feature could be accessed directly by right menu then.